projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.html

```diff
@@ -5,7 +5,7 @@
         {{ title }}
       </div>
 
-      <div class="body" *ngIf="displayBody$ | async">
+      <div class="body" [hidden]="!(displayBody$ | async)">
```

Comment:
Can you think of any reason where we this could be a problem? (not destroying the body of the current panel)

I don't for now but I'm wondering whether it could :thinking:

Reply:
Also, I'm wondering if that's not a "hack" rather than a proper solution regarding the change detection issue.

Eventually something wrong deeper in the code? Any thoughts?

Reply:
Well if we're to extend to animate we need to be hiding not destroying. I had a go at replicating the animate from mat-drawer but the css complexity was way to high to bother with today. I agree that it seems that it shouldn't be required, but given the future requirement of animation I think it's a sensible interim solution